roles/ansible/tower/config-ansible-tower-ocp-ldap-ca/README.md

```diff
@@ -0,0 +1,40 @@
+config-ansible-tower-ocp-ldap-ca
+============================
+
+This role is a helper for `config-ansible-tower-ocp` to create and provision CA certificate secret and ConfigMap which is  used to validate connection with LDAP
+
+## Requirements
+
+  -  running OpenShift Cluster
+  -  'oc' client installed on the Ansible host
+
+
+## Role Variables
+
+The variables used to create and deploy CA secret/ConfigMap  Ansible Tower on OpenShift are outlined in the table below. 
+
+| Variable | Description | Required | Defaults |
+|:---------|:------------|:---------|:---------|
+|openshift_project|OCP project in which Ansible Tower is deployed|yes||
```

Comment:
Make sure this aligns with the way it is written in the `config-ansible-tower-ocp` README - e.g.: "no" for Required, etc.